app/.storybook/main.js

```diff
@@ -50,6 +55,12 @@ module.exports = {
       }),
     ]
 
+    // Workaround for TsconfigPathsPlugin not being able to resolve @public
+    config.resolve.alias = {
+      ...config.resolve.alias,
+      '@public': path.resolve(__dirname, '../public'),
+    }
+
```

Comment:
Notes for reviewers: We use the [tsconfig-paths-webpack-plugin](https://github.com/dividab/tsconfig-paths-webpack-plugin) to pull in path aliases that are defined in `tsconfig.json`. However, it is unable to resolve the `@public` path alias, so I explicitly added this workaround so that the imports from `@public` can resolve in storybook.